modules/show_map.R

```diff
@@ -290,8 +301,8 @@ show_map_server <- function(id, com_module) {
       else{done}
       
       isolate(add_sensors_map())
-      isolate(add_lmls_map())
-      isolate(add_knmi_map())
+      #isolate(add_lmls_map())
```

Comment:
Why are those removed? Are those included in the add_sensors? Or just temporarily?